deploy/crds/mongodb.com_v1_mongodb_scram_cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: mongodb.com/v1
+kind: MongoDB
+metadata:
+  name: example-scram-mongodb
+spec:
+  members: 3
+  type: ReplicaSet
+  version: "4.2.6"
+  security:
+    authentication:
+      enabled: true
```

Comment:
will be updated to remove this option - will always be true

Reply:
what we ended up deciding on was that it will not be possible to disable auth. So the enabled field will be removed alltogether. (currently Authentication is a struct so it can never be nill)

I can remove it in this PR to clear things up

Reply:
no need to remove it. it makes sense